tests/unit/data/test_extract_metadata.py

```diff
@@ -54,42 +54,136 @@ def cube_2():
 
     @staticmethod
     @pytest.fixture
-    def cube_metadata(cube_1):
-        return data.extract_metadata.extract_metadata(cube_1, 1, [], ['cube'], ['netCDF'])
+    def cube_3():
+        latitude = DimCoord(np.linspace(-150, 150, 4),
+                            standard_name='latitude',
+                            units='degrees')
```

Comment:
~~your IDE seems to be using a much shorter line length than everyone else (looks like 80 chars?) Do you think you could change it to 120, which is the default for most modern IDEs~~

Never mind, it seems someone has configured flake8 to use 88 via `setup.cfg`, so it's probably not your editor